app/components/agenda/agenda-header/agenda-actions.js

```diff
@@ -244,7 +244,7 @@ export default class AgendaAgendaHeaderAgendaActions extends Component {
   }
 
   @action
-  async downloadDocuments() {
+  async downloadDocuments(decisions = false) {
```

Comment:
fyi, I hinted at reusing the current logic to prevent duplication and to get a first draft working
Perhaps we could pass a type 'documents' or 'decisions' to make it more generic and expandable (even though there aren't really any other types at the moment)